### PR TITLE
feat(animationproxymanager): add animation manager, and animation proxies

### PR DESCRIPTION
**Pull request by boucaud**
An implementation of animations in the proxy system, with support for the vtkTimeStepBasedAnimationHandler.

The goal is integration into Paraview Glance.

### CODE REVIEW

**Comment by boucaud** [comment]
@jourdain what do you think of the API ?

Also, I'm not sure if an animation manager instance should be directly composed in the ProxyManager, or if they should stay separate. What is you opinion ?

@finetjul FYI

File: Sources/IO/Core/HttpSceneLoader/index.js [hunk 1]
```diff
@@ -307,8 +305,10 @@ function loadTimeStepBasedAnimationHandler(data, model) {
     scene: model.scene,
     originalMetadata: model.metadata,
     applySettings,
-    renderer: model.renderer,
   });
+  if (model.renderer) {
```
Comment (jourdain) [inline Sources/IO/Core/HttpSceneLoader/index.js:309]: Should we have both `model.animationHandler && model.renderer` in the if?

**Comment by jourdain** [comment]
I would create the animation proxy from the proxy manager calling `pxm.create(group, name)`

**Comment by jourdain** [comment]
Looks reasonable otherwise

**Comment by boucaud** [comment]
> I would create the animation proxy from the proxy manager calling `pxm.create(group, name)`

I've updated the example to do it this way.

**Pull request merged**